examples/ssoadmin/managedpolicyattachment.yaml

```
@@ -3,18 +3,18 @@ kind: ManagedPolicyAttachment
 metadata:
   annotations:
     meta.upbound.io/example-id: ssoadmin/v1beta1/managedpolicyattachment
-    upjet.upbound.io/manual-intervention: "This resource requires a valid instanceArn(The Amazon Resource Name (ARN) of the SSO Instance under which 
+    upjet.upbound.io/manual-intervention: "This resource requires a valid instanceArn(The Amazon Resource Name (ARN) of the SSO Instance under which
     the operation will be executed)."
   labels:
-    testing.upbound.io/example-name: example
-  name: example
+    testing.upbound.io/example-name: ssoadmin-managed-policy-attachment
+  name: ssoadmin-managed-policy-attachment
 spec:
   forProvider:
     instanceArn: ${tolist(data.aws_ssoadmin_instances.example.arns)[0]}
```

nit: for consistency, it may be better to put the same value on all examples, either we should adopt other examples to this, or we should adapt these others.

```suggestion
    instanceArn: ARN_OF_MANUALLY_CREATED_SSO_INSTANCE
```

---

I just realized that these examples aren't only used for uptest, they're also shown on the provider docs webpage, where the terraform syntax (which won't work) will be misleading. 

I'll make them all consistent and remove the tf syntax. If someone needs to reference it it's still in examples-generated.